support/cas-server-support-thymeleaf/src/main/resources/templates/delegated-authn/casDelegatedAuthnStopWebflow.html

```diff
@@ -21,8 +21,10 @@ <h2 th:utext="#{screen.pac4j.unauthz.heading}">Unauthorized Access</h2>
 
                 <p>
                     <i class="mdi mdi-alert fas fa-exclamation-triangle mr-2"></i>
-                    <span th:text="#{${'screen.pac4j.authn.' + rootCauseException.class.simpleName}}">
-                        Authentication response provided to CAS by the external identity provider cannot be accepted.</span>
+                    <span th:if="${rootCauseException}" th:text="#{${'screen.pac4j.authn.' + rootCauseException.class.simpleName}}">
+                      Authentication response provided to CAS by the external identity provider cannot be accepted.</span>
+                    <span th:unless="${rootCauseException}" th:text="#{screen.pac4j.authn.unknown}">
```

Comment:
Is `screen.pac4j.authn.unknown` actually available as a message code? If not, you should add this with an explanation.

Reply:
I'm not sure for that. 
Or we just remove `th:text="#{screen.pac4j.authn.unknown}"` ?

Reply:
You should probably add that key to [messages.properties](https://github.com/apereo/cas/blob/master/support/cas-server-support-thymeleaf/src/main/resources/messages.properties#L342). Don't worry about translations in the other bundles. Having it in a bundle lets people deploying CAS override the verbiage in their own message bundle without changing the html file.